oh_staff_ui/views_utils.py

```diff
@@ -84,6 +84,9 @@ def get_edit_item_context(item_id: int) -> dict:
     copyright_formset = get_copyright_formset(item_id)
     language_formset = get_language_formset(item_id)
     resource_formset = get_resource_formset(item_id)
+    top_parent = get_top_parent(item)
```

Comment:
`get_edit_item_context()` doesn't need to know how to assemble what's needed.  I recommend adding a `get_relatives(item)` method which builds `relatives`, which can then be included in the context.  See my other comment about `top_parent` seeming redundant in the context.